packages/tools/webpack/extract-i18n-expressions-webpack-plugin.js

```diff
@@ -1,47 +1,39 @@
-const { sources: { RawSource } } = require( 'webpack' );
+const fs = require( 'fs' );
+const path = require( 'path' );
 
 const MODULE_FILTERS = Object.freeze( [ /(([^!?\s]+?)(?:\.js|\.jsx|\.ts|\.tsx))$/, /^((?!node_modules).)*$/ ] );
 
-module.exports = class ExtractI18nExpressionsWebpackPlugin {
-	translationsRegexps;
-	generateTranslationFilename;
-
-	constructor( {
-		// WordPress i18n function, example for regex match: `__('Hello', 'elementor')`, `_n('Me', 'Us', 2, 'elementor-pro')`.
-		translationsRegexps = [ /\b_(?:_|n|nx|x)\(.*?,\s*(?<c>['"`])[\w-]+\k<c>\)/ ],
-		generateTranslationFilename,
-	} = {} ) {
-		if (
-			! translationsRegexps ||
-			! Array.isArray( translationsRegexps ) ||
-			translationsRegexps.some( ( regexp ) => ! ( regexp instanceof RegExp ) )
-		) {
-			throw new Error( '`translationsRegexps` must be an array of RegExp' );
-		}
+const COMMENTS_REGEXPS = Object.freeze( [
+	// `/* translators: %s is a placeholder for the name of the plugin. */`.
+	/\/\*[\t ]*translators:.*\*\//gm,
+	// `// translators: %s is a placeholder for the name of the plugin.`.
+	/(\/\/)[\t ]*translators:[^\r\n]*/gm,
```

Comment:
those comments are weird to read, maybe add something like "matches: ..." before it?

```suggestion
	// Matches translatiors comment block: `/* translators: %s */`.
	/\/\*[\t ]*translators:.*\*\//gm,
	
	// Matches translators inline comment:  `// translators: %s`.
	/(\/\/)[\t ]*translators:[^\r\n]*/gm,
```

Reply:
Done